app/views/spree/admin/orders/_form.html.haml

```diff
@@ -4,23 +4,8 @@
 
   = render :partial => "spree/admin/orders/shipment", :collection => @order.shipments, :locals => { :order => order }
 
-  - if order.adjustments.eligible.exists?
-    %fieldset.no-border-bottom
-      %legend= Spree.t('adjustments')
-      %table
-        %thead
-          %th= Spree.t('name')
-          %th= Spree.t('amount')
-        %tbody#order-charges.with-border{"data-hook" => "order_details_adjustments"}
-          - order.adjustments.eligible.each do |adjustment|
```

Comment:
The adjustments are no longer limited to those with `eligible: true`.

Reply:
ah, I just verified that adjustment.set_eligibility is removing adjustments with amount zero...
so, this needs to be fixed now. I will create a spree upgrade issue to fix this and the order-charges issue above.
Thanks for you review @kristinalim

Reply:
#3477

Reply:
@luisramos0 I also realized that the Enterprise Fee Summary still needs to be scoped to just `eligible: true`. :grin: Thanks.